### ISSUE TITLE
Version 0.13 Potential Items

### ISSUE THREAD

**Issue by davidslater** [twosixlabs/armory#823]
Pulled from similar 0.12 issue

Prioritized by importance and sequence
    - [ ] Dataset upload/integration tool?
    - [ ] Compression defenses rework?
- [ ] Sanity checking of the model
- [ ] Tagging model output (output format - transcript or prob lattice...; logits vs probs vs log probs)
- [ ] Explicit task / scenario definitions
    - [ ] Use `attack.targeted` as a uniform way of grabbing this from ART to ensure attacks are provided the correct labels.
- [ ] Dataset metadata (speaker gender, input shape, sampling rate / frame rate, number of classes)
- [ ] Integrated model training
    - [ ] Method to save models
    - [ ] Single core issues

Next release?
- [ ] Improved Jupyter integration
- [ ] Improved attack saving/viewing/iteration
- [ ] Update black formatter version
- [ ] Upgrade to newer docker-py
- [ ] Convenience functions for grabbing models/datasets/etc. in a single line from config file

**Comment by davidslater** [comment]
Updated to reflect changes since 0.12
